aws_vpc_policies/aws_vpc_default_network_acl_restricts_all_traffic.py

```diff
@@ -1,11 +1,12 @@
 from panther_base_helpers import IN_PCI_SCOPE  # pylint: disable=import-error
+from panther_oss_helpers import resource_lookup  # pylint: disable=import-error
 
 
 def policy(resource):
     if not IN_PCI_SCOPE(resource):
         return True
 
-    for acl in resource['NetworkAcls']:
-        if acl['IsDefault']:
-            return not acl['Entries']
-    return False
+    default_id = 'arn:aws:ec2:' + resource['Region'] + ':' + resource[
+        'AccountId'] + ':network-acl/' + resource['DefaultNetworkAclId']
```

Comment:
nit: F-Strings may be preferable here e.g.
```
default_id = 
    f"arn:aws:ec2:{resource['Region']}:{resource['AccountId']}:network-acl/{resource['DefaultNetworkAclId']}"```

Reply:
I like it, will update.